### DISCUSSION TITLE
.azureaspireazddevconfig.json doesnt generate on azd up azd deploy

### DISCUSSION THREAD

**Discussion by SalomonHenke** [Azure/azure-dev#3982] [Q&A]
As the title sugests, im unable to find the .azureaspireazddevconfig.json file on deployment. 
![image](https://github.com/Azure/azure-dev/assets/96744651/c503cd93-1fd3-4f60-bc5c-5a9073435d84)
![image](https://github.com/Azure/azure-dev/assets/96744651/9bc21b9d-a4cb-4f24-837b-b65ba7b075da)
https://learn.microsoft.com/en-us/dotnet/aspire/deployment/azure/aca-deployment-azd-in-depth?toc=%2Fazure%2Fdeveloper%2Fazure-developer-cli%2Ftoc.json&bc=%2Fazure%2Fdeveloper%2Fazure-developer-cli%2Fbreadcrumb%2Ftoc.json&tabs=windows

Reason is; I want to preconfigure the Ingress and Cors (which i presume is where i would configure it?)

If my presumption is wrong and the configuration of Ingress and Cors is done differently, then i'm really interested in how. Thanks in advance.

**Comment by rajeshkamal5050** [comment]
Looks like you are using the older bits and docs. Please try with the latest aspire and azd bits/docs. Configuring internal/external for services can be controlled in the AppHost itself - https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/networking-overview#networking-in-the-inner-loop

cc @vhvb1989 @ellismg
